File: II sem/Programowanie Obiektowe/Projekty/projekt2/po3/src/Trawa.java
public class Trawa extends Roslina {

    public Trawa(String ikonka, int x_pos, int y_pos, int sila, int inicjatywa, Swiat swiat) {
        super(ikonka, x_pos, y_pos, sila, inicjatywa, swiat);
    }

    public Trawa(Organizm organizm) {

        super(organizm);
    }

    @Override
    void kolizja(Organizm organizm) {

    }

    @Override
    Organizm kopiuj(Organizm organizm) {
        return new Trawa(organizm);
    }

    @Override
    public void moc(Organizm organizm) {

    }
}

File: II sem/Programowanie Obiektowe/Projekty/projekt1/POproject1/Trawa.cpp
#include "Trawa.h"

Trawa::Trawa(int x, int y) : Roslina(EMOJI ? u8"T" : "Tr", 0, x, y)
{

}

Organizm* Trawa::potomek()
{
	return new Trawa(*this);
}